src/dune_patch/dune_patch.mli

```diff
@@ -0,0 +1,4 @@
+open Stdune
+
+val files_of_patch : dir:Path.t -> string -> Path.t list
+val action : input:Path.t -> Dune_engine.Action.t
```

Comment:
This should return `Dune_engine.Action.t Action_builder.t` with the appropriate dependencies on the input and the patch binary.